test/repository/SchedulesRepository.integration.spec.ts

```diff
@@ -38,12 +40,12 @@ describe('SchedulesRepository', () => {
     });
 
     it('only one schedule is active', async () => {
-      const active = await schedulesRepository.isActiveSchedule();
+      const active = await schedulesRepository.isActiveSchedule(scheduleName);
 
       const notActiveScheduleId = 'not active';
       const notActiveConnection = await Connection.create({ url: mongo.getUri() }, pingInterval, notActiveScheduleId);
       const notActiveSchedulesRepository = notActiveConnection.getSchedulesRepository();
-      const notActive = await notActiveSchedulesRepository.isActiveSchedule();
+      const notActive = await notActiveSchedulesRepository.isActiveSchedule(scheduleName);
```

Comment:
Perhaps rename the variable to something like `isActive`? It's a bit hard to understand with the negation.